cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/schedule/AbstractSchedulerService.java

```diff
@@ -169,6 +171,7 @@ public void resumeSchedule(Id.Program program, SchedulableProgramType programTyp
       scheduler.resumeSchedule(program, programType, scheduleName);
     } catch (NotFoundException e) {
       LOG.trace("Could not resume schedule", e);
+      Throwables.propagate(e);
```

Comment:
The methods of the ``Scheduler`` interface don't throw any exceptions, that's why. I put a comment above, asking if you thought I should modify the interface to better handle errors.